scrapyrt/core.py

```diff
@@ -156,6 +159,7 @@ def crawl(self, *args, **kwargs):
             dfd = self.crawler_process.crawl(self.spider_name, *args, **kwargs)
         except KeyError as e:
             # Spider not found.
+            # TODO is it the only possible exception here?
```

Comment:
it's the only exception that requires 404 response - because spider wasn't found